### ISSUE TITLE
systemctl status on stopped process in handler fails with $? == 3, 2.0.0

### ISSUE THREAD

**Issue by mattwillsher** [ansible/ansible#12216]
```
ansible 2.0.0 (devel 843138e7d9) last updated 2015/09/02 18:58:01 (GMT +100)
  lib/ansible/modules/core: (detached HEAD bbcfb1092a) last updated 2015/09/02 19:18:09 (GMT +100)
  lib/ansible/modules/extras: (detached HEAD b8803306d1) last updated 2015/09/02 19:18:09 (GMT +100)
  config file = /Users/matt/Development/ansible/ansible.cfg
  configured module search path = None
```

``` yaml

---
  - name: ntpd_restart
    service:
      name: ntpd
      state: restarted
      enabled: True
```

```
RUNNING HANDLER [ntpd : ntpd_restart] ******************************************
fatal: [testvm]: FAILED! => {"changed": false, "failed": true, "msg": "failure 3 running systemctl status for 'ntpd': "}
```

`systemctl status ntpd` has $? == 3 when ntpd is stopped

```
[root@testvm ~]# systemctl status ntpd
ntpd.service - Network Time Service
   Loaded: loaded (/usr/lib/systemd/system/ntpd.service; enabled)
   Active: inactive (dead) since Wed 2015-09-02 19:34:27 BST; 3min 16s ago
  Process: 21871 ExecStart=/usr/sbin/ntpd -u ntp:ntp $OPTIONS (code=exited, status=0/SUCCESS)
 Main PID: 21872 (code=exited, status=0/SUCCESS)
....
[root@testvm ~]# echo $?
3
```

I was expecting this error to be ignored and the service started by the handler.

**Comment by bcoca** [comment]
it seems i made a bad assumption in my last fix for the lack of errors of 'show'

**Comment by bcoca** [comment]
i did not realize it, but his is in wrong repo (service module is in github.com/ansible/ansible-modues-core). This should be fixed by my commit e278f285aa6f61e45416be28b1e689b4d7607196 in that repo.
